### ISSUE TITLE
py-ipfs hangout

### ISSUE THREAD

**Issue by amstocker** [ipfs/pm#34]
Just curious; who would be interested in a python ipfs hangout as part of the monday sprint?  Here are some possible topics of discussion:

- python-ipfs-api
- IPFS extensions for flask, django, etc
- possible full implementation (or partial implementation) of ipfs in python
- distributed python applications built on ipfs
- other things??

**Comment by jbenet** [comment]
Maybe tag the relevant people here, as most people won't see this



—

**Comment by RichardLitt** [comment]
Also maybe cross-ref it in IPFS? There might be some Pythonistas who are only following that repo who might be keen to help.

**Comment by amstocker** [comment]
Thanks for the suggestions, I'll get on it once i get home

**Comment by RichardLitt** [comment]
@amstocker Word on this?

**Comment by amstocker** [comment]
@RichardLitt I'm pretty busy with school at the moment, but I could probably start organizing this for the sprint starting in a few weeks maybe?

**Comment by RichardLitt** [comment]
@amstocker That works. I would cross-ref to ipfs/ipfs; we don't know if anyone else is interested yet, AFAIK.

**Comment by amstocker** [comment]
@RichardLitt [made a new issue](https://github.com/ipfs/ipfs/issues/112) on ipfs/ipfs, so we'll see who comes out of the woodwork.

**Comment by RichardLitt** [comment]
Nice! Let's.

**Comment by tteresaann1nonly** [comment]
Thanks, I'm still learning, but coming along swell!!!!

**Comment by RichardLitt** [comment]
Let's wait on that issue, and close this one until there is more interest.
